kinto/core/resource/__init__.py

```diff
@@ -599,7 +613,7 @@ def process_record(self, new, old=None):
 
         return new
 
-    def apply_changes(self, record, changes):
+    def apply_changes(self, record, changes, operations=None):
```

Comment:
I think we should do it for all calls for consistency. This also allows resources to support their own modification syntaxes, although I guess that nobody will actually do that. So I'm proposing the signature `def apply_changes(self: Resource, record: Record, requested_changes: object) -> (updated: Record, applied_changes: {FieldName: Value})`. (Or maybe `applied_changes` should just be `[FieldName]`?)